tpkg/ixfr_retry.tdir/ixfr_retry.datafile

```diff
@@ -0,0 +1,65 @@
+$ORIGIN bar.foo.
+$TTL 7200
+
+ENTRY_BEGIN
+MATCH opcode qtype qname serial=1
+REPLY QUERY
+REPLY NOERROR
+REPLY AA AD
+ADJUST copy_id
+SECTION QUESTION
+baz.foo. IN IXFR
+SECTION ANSWER
+baz.foo. IN SOA    ns.baz.foo. hostmaster.baz.foo. 2 300 4 3000 5
+; deleted items
+baz.foo. IN SOA    ns.baz.foo. hostmaster.baz.foo. 1 3   4 3000 5
+baz.foo. IN SOA    ns.baz.foo. hostmaster.baz.foo. 2 300 4 3000 5
+; added items
+web1     IN A      4.5.6.7
+web2     IN A      5.6.7.8
```

Comment:
The extension of web1 and web2 is bar.foo, but the IXFR is listed as baz.foo.  See the `$ORIGIN` on line 1.  And on line 24, and the other entries.